demo/server/src/modules/keycloak/index.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path="./keycloak.d.ts" />
+
+import * as express from 'express';
+import * as session from 'express-session';
+import Keycloak = require('keycloak-connect');
```

Comment:
Does `import * as Keycloak from 'keycloak-connect'` work when you have the d.ts file?

I think requires are a tslint error atm, but feel free to ignore the rule with a comment if required.